src/data/alumni.ts

```diff
@@ -0,0 +1,127 @@
+export interface Alumni {
+  name: string;
+  description: string;
+}
+
+export const alumni: Alumni[] = [
+  {
+    name: 'Khanh Nguyen',
+    description: `Khanh is a Junior from Vietnam studying Computer Science. BROCKHAMPTON is his all-time favorite boyband. Outside of school, he enjoys trying out good food, exploring the city and contributing to projects. 
+    Khanh was the former president.`,
```

Comment:
```suggestion
    description: `Khanh is an alumni from Vietnam majoring in Computer Science. BROCKHAMPTON is his all-time favorite boyband. Outside of school, he enjoys trying out good food, exploring the city and contributing to projects. Khanh was a former president.`,
```
Keep consistency and have all data on one line. The rendering does not seem to preserve the newlines anyway.

Also remove reference to their university year since it is no longer relevant or accurate.